### ISSUE TITLE
vca.py from Module Utils throws an error, when using vca_nat Plugin: 'VCD_REQ_ARGS' is not defined

### ISSUE THREAD

**Issue by ruslantum** [ansible/ansible#18584]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
Module Utils / vca.py

##### ANSIBLE VERSION
```
ansible 2.2.0.0
  config file = 
  configured module search path = Default w/o overrides
```

##### OS / ENVIRONMENT

N/A

##### SUMMARY

When using the vca_nat Plugin to create NAT Rules in vCloud Director with "vcd" as a service_type, following error happens: NameError: global name 'VCD_REQ_ARGS' is not defined

##### ACTUAL RESULTS

```
Traceback (most recent call last):\
  File \"/tmp/ansible_BiGD9q/ansible_module_vca_nat.py\", line 215, in <module>\
    main()\
  File \"/tmp/ansible_BiGD9q/ansible_module_vca_nat.py\", line 148, in main\
    vca = vca_login(module)\
  File \"/tmp/ansible_BiGD9q/ansible_modlib.zip/ansible/module_utils/vca.py\", line 286, in vca_login\
  File \"/tmp/ansible_BiGD9q/ansible_modlib.zip/ansible/module_utils/vca.py\", line 228, in _validate_module\
NameError: global name 'VCD_REQ_ARGS' is not defined
```

##### POSSIBLE FIX
```
diff vca.py vca_old.py
207d206
< VCD_REQ_ARGS = []
```

**Comment by gundalow** [comment]
@ruslantum That seems like a sensible fix, would you like to check that works for you and if so raise a PR to fix it?

**Comment by ruslantum** [comment]
@gundalow Hey, yeah that fix works for me, and I created the pr now:
https://github.com/ansible/ansible/pull/18718

**Comment by gundalow** [comment]
Fixed by #18718
